dapr/proto/runtime/v1/dapr.proto

```diff
@@ -600,7 +598,12 @@ message InvokeActorResponse {
   bytes data = 1;
 }
 
-// GetMetadataResponse is a message that is returned on GetMetadata rpc call
+// GetMetadataRequest is the message for the GetMetadata request.
+message GetMetadataRequest {
```

Comment:
Yes, but:

1. This will not break apps that are currently running, which is the most important thing. Users may need to change a struct name if they import the protos directly and are updating the protos.
2. Using the gRPC protos directly is not an officially supported scenario.

Reply:
> Using the gRPC protos directly is not an officially supported scenario.

Still something that users might use. Again, my point being here is that it must be communicated as a breaking change. We can expand on it saying that running apps will not be affected, just struct change as of now for new proto imports.

Reply:
This is a breaking change, however this will be breaking at compilation time and not at runtime. I recommend we announce the breaking change with no need to create a v2 of this method.

Reply:
>  This is a breaking change, however this will be breaking at compilation time and not at runtime. I recommend we announce the breaking change with no need to create a v2 of this method.

This is a fair assessment.

To recap, and to be fully clear, from a practical effect this change is similar to what happened with #6945 (landing in 1.13 too) which removed a method from the gRPC proto (of course, that was done for different reasons).

- No impact on apps currently running
- Dapr SDKs maintainers will need to rename the structs/objects when they upgrade the protos
- Users of the SDKs should not see any direct impact.
  - AFAIK the SDKs always abstract the gRPC objects, so users who just depend on Dapr SDKs shouldn't need any code change if/when they upgrade the SDKs.
  - At runtime, old SDKs will continue to work alongside new SDKs.
- For users who import the protos directly:
   - No change needed if they are using existing protos
   - If they upgrade the protos, they will need to rename the structs/objects